docs/UserGuide.md

```diff
@@ -55,7 +55,7 @@ No tasks to display.
 | Command |Format |
 | --- | --- |
 |add|`add <task_name> [st/<start> end/<end> | dl/<deadline>] [#<tags>]  [r/<recurrence>] [desc/<description>]`|
-|list|`list [[[after/<date>] [before/<date>] | [on/<date>]][tags=<comma_separated_tags>] [recurrence=<recurrence_value>] [desc=<description_value>]]`|
+|list|`list [[keywords] [[after/<date>] [before/<date>] | [on/<date>]][#<tags>] [recurrence=<recurrence_value>] [desc=<description_value>]]`|
```

Comment:
Should we use [#<tag_name> …] to keep it consistent with tag/untag?
The current format may be interpreted as #tag1,tag2